modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -668,18 +668,18 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::computeTileWeight(
           else
           {
             // Set input to edge of limit so it is found later in computePartitionWeights
-            input = _input_limits[p][t][in_index][0];
+            input = _input_limits[p][t][in_index][1];
             weights[p][t] = 0.0;
             std::stringstream msg;
             msg << "In " << _name << ": input parameter with value ("
                 << MetaPhysicL::raw_value(input) << ") is out of upper global range ("
```

Comment:
print the input before it was reset to limit:
```
msg << "In " << _name << ": input parameter with value ("
                << input_before_set_to_limit << ") is out of upper global range ("
                << _input_limits[p][t][in_index][1] << ")";
```

Reply:
Good catch!

Reply:
Do the same for the lower limit: lines 643 and 639 (for some reason I don't get the little + symbol to add a comment in those lines?)